packages/workbox-runtime-caching/CacheOnly.mjs

```diff
@@ -14,13 +14,14 @@
 */
 
 import {_private} from 'workbox-core';
+import core from 'workbox-core';
 
 /**
- * An implementation of a [cache-only](https://developers.google.com/web/fundamentals/instant-and-offline/offline-cookbook/#cache-only)
+ * An implementation of a
+ * [cache-only]{@link https://developers.google.com/web/fundamentals/instant-and-offline/offline-cookbook/#cache-only}
  * request strategy.
  *
- * This class is useful if you want to take advantage of Workbox cache naming
- * and / or plugins.
+ * This class is useful if you want to take advantage of the Workbox plugins.
```

Comment:
It reads like the previous text here (cache naming/or plugins) was more specific. Was there a reason we wanted the wording more general?

Reply:
IMO the cache naming wasnt a sell to be honest. its trivial to access now.

Reply:
@jeffposnick @philipwalton do you have a preference here?

Reply:
Leaving out mention of the cache names is fine. I'm 👍 with the text in your PR.